### ISSUE TITLE
Nym wallet 1.2.13 doesn't work after updating to Ubuntu 24.04

### ISSUE THREAD

**Issue by KonstantinLitvin** [nymtech/nym#4688]
Run Nym wallet and have an empty field without UI

**Stack Traces**

In:
> ./nym-wallet_1.2.13_amd64.AppImage 

Out:
> /usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
././/lib/x86_64-linux-gnu/webkit2gtk-4.0/WebKitWebProcess: symbol lookup error: /lib/x86_64-linux-gnu/libVkLayer_MESA_device_select.so: undefined symbol: wl_proxy_marshal_flags

**Screenshots**
![image](https://github.com/nymtech/nym/assets/9830255/8ed5ae9c-3d4e-496c-9775-c316791bc13b)

**Which area of Nym were you using?**
 - Wallet
 - Ubuntu 24.04, Wayland

**Comment by gyrusdentatus** [comment]
makes me want to try ubuntu 2404 just to try to replicate this :)

**Comment by KonstantinLitvin** [comment]
Running Nym wallet as root user helped me to bypass this issue

**Comment by rachyandco** [comment]
is it still the case with latest wallet ? https://github.com/nymtech/nym/releases/tag/nym-wallet-v1.2.15

**Comment by KonstantinLitvin** [comment]
> is it still the case with latest wallet ? https://github.com/nymtech/nym/releases/tag/nym-wallet-v1.2.15

Yes, I just checked Nym Wallet Version 1.2.15, and it works only if I run it as root user.

**Comment by rachyandco** [comment]
thanks. it is possibly due to wayland. we will check this

**Comment by gyrusdentatus** [comment]
> Running Nym wallet as root user helped me to bypass this issue

You sir, are brave. :}

**Comment by gnd** [comment]
This is still unsolved as of 1.2.24. @rachyandco

**Comment by KonstantinLitvin** [comment]
I had the same issue on Ubuntu 26.04, it works with the following env var without root  `LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libwayland-client.so ./NymWallet_1.2.23_amd64.AppImage`
